specification/recoveryservicessiterecovery/resource-manager/Microsoft.RecoveryServices/stable/2023-08-01/service.json

```diff
@@ -8548,6 +8606,18 @@
         "recoveryCapacityReservationGroupId": {
           "description": "The recovery capacity reservation group Id.",
           "type": "string"
+        },
+        "autoProtectionOfDataDisk": {
```

Comment:
<nit> dataDiskAutoProtection may be better

Reply:
It is not possible to change this now, also this field name was approved by PMs.